File: Arcade/Intro/Ruby/sortByHeight.rb
def sortByHeight(a)
    a.size.times do |i|
		(i..(a.size - 1)).each do |j|
			if a[i] != -1 && a[j] != -1
                if a[i] > a[j]
                    tmp = a[i]
                    a[i] = a[j]
                    a[j] = tmp
                end
            end
		end
	end
	
	return a
end

File: Arcade/Intro/javascript/sortByHeight.js
function sortByHeight(a) {
    for(let i = 0; i < a.length; i++) {
        for(let j = i + 1; j < a.length; j++) {
            if(a[i] != -1 && a[j] != -1) {
                if(a[i] > a[j]) {
                    let tmp = a[i];
                    a[i] = a[j];
                    a[j] = tmp;
                }
            }
        }
    }
    
    return a;
}